__tests__/authkit-callback-route.spec.ts

```diff
@@ -232,5 +232,30 @@ describe('authkit-callback-route', () => {
 
       expect(response.status).toBe(500);
     });
+
+    it('should call onSuccess if provided', async () => {
+      const mockAuthResponse = {
+        accessToken: 'access123',
+        refreshToken: 'refresh123',
+        user: { id: 'user_123' },
+        oauthTokens: {
+          access_token: 'access123',
+          refresh_token: 'refresh123',
+          expires_at: 1719811200,
+          scopes: ['foo', 'bar'],
+        },
+      };
+
+      (workos.userManagement.authenticateWithCode as jest.Mock).mockResolvedValue(mockAuthResponse);
```

Comment:
[`jest.mocked`](https://jestjs.io/docs/mock-function-api#jestmockedsource-options) was intended to help avoid this type of casting and also retain the type information from the mocked method.

```suggestion
      jest.mocked(workos.userManagement.authenticateWithCode).mockResolvedValue(mockAuthResponse);
```